jetbrains-ultimate/it/software/aws/toolkits/jetbrains/services/lambda/nodejs/NodeJsLocalLambdaRunConfigurationIntegrationTest.kt

```diff
@@ -241,4 +250,21 @@ class NodeJsLocalLambdaRunConfigurationIntegrationTest(private val runtime: Runt
         expectedOutput = input.toUpperCase(),
         addBreakpoint = { projectRule.addBreakpoint() }
     )
+
+    @Test
+    fun `does not timeout if SAM has output`() {
```

Comment:
If we don't have any runtime specific code now, do we actually need to run the test for every runtime?